app/controllers/bookings_controller.rb

```diff
@@ -17,6 +17,7 @@ def seeker_index
     @pending_bookings = Booking.where(user_id: @user, status: "Pending")
     @accepted_bookings = Booking.where(user_id: @user, status: "Accepted")
     @declined_bookings = Booking.where(user_id: @user, status: "Declined")
+    @booking_order = Booking.order(created_at: :desc)
```

Comment:
I'm not sure if this variable is ever used?

Reply:
Not used!